src/types.ts

```diff
@@ -131,10 +146,11 @@ export interface MarketsRow {
   tag2: string|null;
   volume: string|number;
   sharesOutstanding: string|number;
+  marketStateID: number;
   reportingWindow: Address;
   endTime: number;
   finalizationTime?: number|null;
-  reportingState?: number|null;
+  reportingState?: string|null;
```

Comment:
Should use the enum type here: `reportingState?: REPORTING_STATE | null`

Reply:
Totally